torch/csrc/autograd/input_buffer.cpp

```diff
@@ -137,31 +137,32 @@ void InputBuffer::add(
 
   // Switches to accumulate device
   // The device (and stream) chosen for accumulation is:
-  //  (1) var is not a CUDA/privateuse1 variable. Accumulation happens on var's
-  //  device. (2) var is a CUDA/privateuse1 variable and it, the consumer, and
-  //  the producer share the same device:
+  //  (1) var is not a CUDA/XPU/privateuse1 variable. Accumulation happens on
```

Comment:
@majing921201 We can use accelerator to refine these comments as well.

Reply:
sure, thanks